src/libtiled/tilesetformat.h

```diff
@@ -56,14 +57,38 @@ class TILEDSHARED_EXPORT TilesetFormat : public FileFormat
     /**
      * Writes the given \a tileset based on the suggested \a fileName.
      *
-     * This function may write to a different file name or may even write to
-     * multiple files. The actual files that will be written to can be
-     * determined by calling outputFiles().
-     *
      * @return <code>true</code> on success, <code>false</code> when an error
      *         occurred. The error can be retrieved by errorString().
      */
     virtual bool write(const Tileset &tileset, const QString &fileName) = 0;
+
+private:
+    QPointer<TilesetFormat> mExportFormat;
+};
+
+} // namespace Tiled
+
+Q_DECLARE_INTERFACE(Tiled::TilesetFormat, "org.mapeditor.TilesetFormat")
+Q_DECLARE_OPERATORS_FOR_FLAGS(Tiled::FileFormat::Capabilities)
```

Comment:
Please move this operators macro to `fileformat.h`.

Reply:
Done